src/main/kotlin/blackjack/domain/Card.kt

```diff
@@ -0,0 +1,58 @@
+package blackjack.domain
+
+class Card private constructor(
+    val denomination: Denomination,
+    private val suit: Suit
+) {
+
+    override fun toString(): String {
+        return "${denomination.displayedName}${suit.type}"
+    }
+
+    companion object {
+        private val cardsByDenominationAndSuit: Map<Pair<Denomination, Suit>, Card> = init()
+        internal val entireCards: List<Card> = cardsByDenominationAndSuit.values.toList()
+        private fun init(): Map<Pair<Denomination, Suit>, Card> {
+            val map = mutableMapOf<Pair<Denomination, Suit>, Card>()
+            Denomination.values().forEach { denomination ->
+                Suit.values().forEach { suit ->
+                    map[denomination to suit] = Card(denomination, suit)
+                }
+            }
+            return map.toMap()
+        }
+
+        fun from(denomination: Denomination, suit: Suit): Card {
+            return cardsByDenominationAndSuit[denomination to suit] ?: throw IllegalArgumentException()
+        }
+    }
+}
+
+enum class Suit(
+    val type: String
+) {
+    SPADE("스페이드"),
+    HEART("하트"),
+    CLOVER("클로버"),
+    DIAMOND("다이아")
+}
+
+enum class Denomination(
+    val minValue: Int,
+    val maxValue: Int,
+    val displayedName: String
+) {
+    ACE(1, 11, "1"),
```

Comment:
```suggestion
    ACE(1, 11, "A"),
```
Ace의 출력은 A로 표현하고 있어요.

Reply:
앗 이 부분을 제대로 확인 못했네요 ㅠㅠ